devlab/tests/generate_load.py

```diff
@@ -557,7 +557,53 @@ def create_routers(self):
         for router in self.config.routers:
             self.neutronclient.create_router(router)
 
-    @clean_if_exists
+    def get_subnet_id(self, name):
+        subs = self.neutronclient.list_subnets()['subnets']
+        for sub in subs:
+            if sub['name'] == name:
```

Comment:
It is generate_load.py file so we are building the network themselves with different names.